site/sfguides/src/build_embedded_analytics_with_thoughtspot/build_embedded_analytics_with_thoughtspot.md

```diff
@@ -0,0 +1,240 @@
+author: Ali Alladin (ali.alladin@thoughtspot.com)
+id: ai_powered_embedded_analytics_using_thoughtspot
+summary: Guide to create your own embedded analytics application using ThoughtSpot
+categories: Getting-Started, Embedded Analytics, BI
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Science, Data Engineering, analytics, app-development, solution-examples, marketing, retail 
+
+# AI Powered Embedded Analytics using ThoughtSpot
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
+
+When combined with Snowflake’s robust and efficient data cloud solution, ThoughtSpot can help you turn your data into actionable insights. Our AI-Powered Analytics platform is designed for everyone—whether you’re code first or code free. Instead of relying on data teams to crank out clunky dashboards and outdated reports, business users can use a familiar natural language search interface and augmented analytics to discover insights, create and share Liveboards, and make real-time decisions. This frees analysts and data teams to work on more impactful projects, like data pipeline optimizations or complex analysis. By choosing ThoughtSpot and Snowflake, you’re investing in a modern data infrastructure built to scale, grow, and innovate with your business. 
+
+![Product Architecture](assets/ThoughtSpot_arch.png)
+
+### Prerequisites
+- Familiarity with Snowflake and React.
+- ThoughtSpot Free Trial.
+- A Snowflake Account to connect ThoughtSpot
```

Comment:
@aalladin @sfc-gh-lambrosetti Add link to Snowflake signin/signup page.

Reply:
done